src/bindist/autoemu.bat

```diff
@@ -1,8 +1,10 @@
 @echo off
 rem generic autoexec.bat for DOSEMU + any DOS
 rem must be edited in most circumstances
-path c:\dos;c:\windows\command;e:\dosemu
-lredir d: linux\fs\${DOSEMU_LIB_DIR}/freedos ro
+set PATH=c:\windows\command;e:\dosemu

```

Comment:
Yes, the installer can store them somewhere,
but perhaps it would be better if the installer
to adjust the autoexec before installing it?
We can add templates to autoexec file, like
@DOS_DIR@ and installer can replace them
with the values of the similar variables.